designs/gpu-selection-and-bootstrap.md

```diff
@@ -108,10 +109,10 @@ Here are some relevant labels:
 
 | Label                                 | Description                               |
 |---------------------------------------|-------------------------------------------|
-| `karpenter.k8s.azure/sku-family`       | Family of the SKU (e.g., standardAv2Family)|
-| `karpenter.k8s.azure/sku-subfamily`    | Subfamily of the SKU                      |
-| `karpenter.k8s.azure/sku-cpu`          | Number of virtual CPUs                    |
-| `karpenter.k8s.azure/sku-accelerator`  | Type of accelerator (e.g., Nvidia)        |
+| `karpenter.azure.com/sku-family`       | Family of the SKU (e.g., standardAv2Family)|
```

Comment:
unrelated changes?

Reply:
yeah, just figured, i should update these gpu docs while im here

Reply:
`sku-family` value is just a single letter, e.g. "D"

Reply:
good catch